src/allmydata/client.py

```diff
@@ -512,6 +519,14 @@ def tub_creator(handler_overrides=None, **kwargs):
             **kwargs
         )
 
+    # we don't actually use this keypair for anything (yet) as far
+    # as I can see.
+    # my_pubkey = keyutil.parse_pubkey(
+    #     self.get_config_from_file("node.pubkey")
+    # )
```

Comment:
Maybe we can lose this comment now.  Also I noticed that in `_Client.init_node_key` we rederive the node public key and then re-write it over the existing node.pubkey - we never actually read the value back.  This is nice since it means your pubkey and privkey are always consistent with each other.  It also means the node.pubkey file doesn't necessarily even exist at this point, I guess.